Tools/scripts/build_binaries.py

```diff
@@ -300,6 +300,11 @@ def touch_filepath(self, filepath):
             with open(filepath, "a"):
                 pass
 
+    def board_is_chibios(self, board):
+        if board == 'skyviper-v2450':
```

Comment:
I'm hoping we can move compilers forward for everything before it becomes an issue!

Reply:
I have my doubts we will be able to do that soon, because later versions of GCC increased the flash space with PX4, sufficiently that v2 would be out of space - and we are already running into that problem frequently now.

On the other hand, it's quite likely we'll soon want to have automated builds of ChibiOS so that more people can try it.

Reply:
In fact, in was true for gcc5 but not for gcc7! see on discuss the output of build for different targets and compiler !

Reply:
Oh that's good to know! ChibiOS is using GCC 6 though and last time I checked (quite a few months ago) it was giving larger binaries.